bundle/manifests/components.platform.opendatahub.io_workbenches.yaml

```diff
@@ -74,6 +74,13 @@ spec:
                       type: object
                     type: array
                 type: object
+              workbenchNamespace:
+                default: rhods-notebooks
```

Comment:
We have this value configured in our OdhDashboardConfig
https://github.com/opendatahub-io/odh-dashboard/blob/main/manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml#L41

We'll need something akin to the "Auth" resource treatment where we migrate the effort away from our resource and into Platform's hand.

Which was part of the RHOAISTRAT-419 work done in 2.17. cc @StevenTobin @lphiri